src/pages/concept-registration-page/form-term/form-term.component.tsx

```diff
@@ -30,7 +30,9 @@ const config = {
     dispatch: dispatch
   }),
   validationSchema: schema,
-  validate: _.throttle(patchConceptFromForm, 250),
+  validate: _.throttle((values, { concept, dispatch }) => {
+    validateConceptForm(values, schema, concept, dispatch);
```

Comment:
flere spørsmål her:
- er schema dynamisk? hvis ikke, da kan du kanskje laste det inni validasjonsfunksjon, og slippe å ha det som argument.
- hvem kaller validate? kandet ikke også gi schema (så du kan unngå å mappe argumenter)
- trenger ikke { for en enkel statement, og samtidlig, du glemte å return (eller var det mening)
- er det validate eller async validate?

Reply:
Svarene er gitt av Formik som vi bruker nå, ikke redux-form.